### ISSUE TITLE
support iterable as children

### ISSUE THREAD

**Issue by futurist** [foxdonut/seview#2]
In react iterable as children is valid:

```jsx
    const element1 = (
      <React.Fragment>
        {new Set([<span key="a">hi</span>, <span key="b">bye</span>])}
      </React.Fragment>
    );
```

should review support iterable instead of only Array?

**Comment by foxdonut** [comment]
@futurist that's a good question. What is the use case for using `new Set([...])` instead of just `[...]`?

Of course you can always use `Array.from(set)` as well, but I'm curious about why you'd have a Set of view nodes in the first place.

**Comment by futurist** [comment]
I want to go a through test for seview, plan to write a script to replace all JSX in __tests__  files into seview, until found below line:

https://github.com/facebook/react/blob/2a2ef7e0fd86612404676b15bb3dd9df7ade536e/packages/react-reconciler/src/__tests__/ReactFragment-test.js#L74

Get stuck for `new Set`, and wonder should seview totally compatible with React?

The `Array.from` is good but have problem when the children is like:

```js
<Component>{this.props.children}</Component>
```

That's the root of this issue, if this issue solved then maybe a babel-plugin can be good as a converter.

**Comment by foxdonut** [comment]
@futurist fixed in version 0.8.0
